virtio-devices/src/vhost_user/mod.rs

```diff
@@ -127,8 +126,9 @@ pub const DEFAULT_VIRTIO_FEATURES: u64 = 1 << VIRTIO_F_RING_INDIRECT_DESC
     | VhostUserVirtioFeatures::PROTOCOL_FEATURES.bits();
 
 const HUP_CONNECTION_EVENT: u16 = EPOLL_HELPER_EVENT_LAST + 1;
+const SLAVE_REQ_EVENT: u16 = EPOLL_HELPER_EVENT_LAST + 2;
 
-pub struct ReconnectEpollHandler {
+pub struct ReconnectEpollHandler<S: VhostUserMasterReqHandler> {
```

Comment:
I think the name of `ReconnectEpollHandler` should be more generic, as it is now also handling the slave requests.

Reply:
I agree, I'll add a new commit to take care of this.